tests/DCPS/WriteDataContainer/WriteDataContainerTest.cpp

```diff
@@ -350,6 +350,8 @@ int run_test(int argc, ACE_TCHAR *argv[])
 
   try
     {
+    DDS::DomainParticipantFactory_var dpf =
```

Comment:
Yes, this is needed else we don't initialize the service participant and all kinds of mandatory members are not set causing crashes